From 6c1340e2dc945c83a9a564c884b96390830bbb1f Mon Sep 17 00:00:00 2001
From: Roel Janssen <roel@gnu.org>
Date: Mon, 16 Apr 2018 10:31:31 +0200
Subject: [PATCH] Properly detect and run fragment mode.

* templates/Map.sh.tt: Use the SINGLE_END variable instead of fastq.R2 (which is
  always defined).
---
 templates/Map.sh.tt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/templates/Map.sh.tt b/templates/Map.sh.tt
index 19539d6..f9a43dc 100644
--- a/share/hmf-pipeline/templates/Map.sh.tt
+++ b/share/hmf-pipeline/templates/Map.sh.tt
@@ -12,7 +12,7 @@ cd "[% dirs.mapping %]"
     -R "@RG\tID:[% fastq.coreName %]\tSM:[% fastq.sampleName %]\tPL:ILLUMINA\tLB:[% fastq.sampleName %]\tPU:[% fastq.flowcellID %]" \
     "[% opt.GENOME %]" \
     "[% fastq.R1 %]" \
-    [%- IF fastq.exists('R2') %]
+    [%- IF ! opt.exists('SINGLE_END') %]
     "[% fastq.R2 %]" \
     [%- END %]
     | [% opt.SAMBAMBA_PATH %]/sambamba view \
-- 
2.17.0

